velox/docs/develop/scalar-functions.rst

```diff
@@ -581,14 +581,16 @@ process. This set may not include all the rows. By default, a vector function
 is assumed to have the default null behavior, e.g. null in any input produces
 a null result. In this case, the expression evaluation engine will exclude
 rows with nulls from the “rows” specified in the call to “apply”. If a
-function has a different behavior for null inputs, it must override the
-isDefaultNullBehavior method to return false.
+function has a different behavior for null inputs, VectorFunctionMetadata's
+defaultNullBehavior must be set to false when it is registered.
 
 .. code-block:: c++
 
-    bool isDefaultNullBehavior() const override {
-      return false;
-    }
+    exec::registerStatefulVectorFunction(
```

Comment:
The Registration section that talks about registration comes later. Hence, this code here will come as a surprise to the reader.

Maybe we could update the Registration section to clarify that registerXxx APIs take an optional 'metadata' parameter that allows to specify xyz behaviors. Then, update the text here to refer to the Registration section.

```
If a function has a different behavior for null inputs, it must specify so during registration.
```

Reply:
@mbasmanova, thanks for your suggestion! It makes sense. I just updated the pr. Please check again.